src/VisualStudio/Core/Def/Implementation/TodoComments/VisualStudioTodoCommentsService.cs

```diff
@@ -221,5 +221,9 @@ public async Task ReportTodoCommentDataAsync(DocumentId documentId, ImmutableArr
             await ReportTodoCommentDataAsync(
                 document.Id, converted.ToImmutable(), cancellationToken).ConfigureAwait(false);
         }
+
+        /// <inheritdoc cref="IVsTypeScriptTodoCommentService.OnDocumentRemovedAsync(DocumentId, CancellationToken)"/>
```

Comment:
📝 This is automatic if you want to remove it

Reply:
Removed here and above. Thanks!